### ISSUE TITLE
shows error when import 

### ISSUE THREAD

**Issue by studyquant** [joliveros/bitmex-websocket#12]
shows error when import 
```
import websocket

from bitmex_websocket import Instrument
from bitmex_websocket.constants import InstrumentChannels


  File "C:\Anaconda3.5\lib\site-packages\bitmex_websocket\_instrument.py", line 43
    channel_key = f'{channel.name}:{self.symbol}'
                                                ^
SyntaxError: invalid syntax
```

**Comment by joliveros** [comment]
Hi @studyquant 
what version of python are you using?

**Comment by studyquant** [comment]
@joliveros @joliveros Python 3.5

**Comment by joliveros** [comment]
Hi @studyquant ,
definitely sounds like this is due to the fact that this module uses f-string which are only available in Python 3.6+

Let me know if you have any other issues.
